docs/_articles/en/user-guide/development-models.md

```diff
@@ -33,7 +33,7 @@ To create developing against non-source-tracked orgs:
 
 ![Create project](./images/create-project-with-manifest.png)
 
-1. In the Side Bar of the code editor, click Org Picker. This opens the Command Palette and you can select the command to authorize an org, Dev Hub, or create a scratch org; Or you can select from the list of authorized orgs.
+1. In the code editor's status bar, click Org Picker to open the Command Palette. Select the command to authorize an org, Dev Hub, or create a scratch org; Or you can select from the list of authorized orgs.
```

Comment:
```suggestion
1. In the code editor's status bar, click Org Picker to open the Command Palette. You can select from the list of authorized orgs, or you can choose to authorize a new org.
```

Reply:
Hey, I just looked up how MS spells "Command Palette" - and they init cap it like it is here. I noticed (and it's partially my fault) that we are inconsistent in the VS Code docs. Not urgent but would be nice to fix all l/c references when articles are updated.